repository-data/webfiles/src/main/resources/site/freemarker/include/cookie-button.ftl

```diff
@@ -0,0 +1,7 @@
+<div class="nhsuk__cookieStatus">
```

Comment:
Looks like `<@fmt.message key="..."/>` calls here are working as the bundles are already loaded on `/freemarker/hee/macros/last-next-reviewed-date.ftl`, but it is good to add the following imports just for the safer side.
```
<#assign fmt=JspTaglibs ["http://java.sun.com/jsp/jstl/fmt"] >
<@hst.setBundle basename="uk.nhs.hee.web.global"/>
```

Reply:
Thanks for pushing this. Checked and works fine.